trillian/migrillian/configpb/config.pb.go

```diff
@@ -15,8 +15,8 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.26.0
-// 	protoc        v3.12.4
-// source: config.proto
+// 	protoc        v3.11.4
+// source: migrillian/configpb/config.proto
```

Comment:
Can you update your tooling? This diff shouldn't be here, unless I'm missing something fundamental about your approach here (in which case, it should be in the PR desc):